### PR TITLE
Add GitHub action workflow for releasing

### PR DESCRIPTION
**Pull request by enkatsu**
Hi @mingness , I have developed a workflow for releasing. This workflow is triggered by pushing a tag in Git. The libraries will be distributed as shown below. Please let me know if you have any feedback 🙏 

https://github.com/enkatsu/processing-library-template/releases/tag/v0.0.1

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-version: 8.5
+      - name: Build with Gradle
+        id: build
+        run: |
+          gradle packageRelease
+          echo zip_path=`find ./release/*.zip` >> $GITHUB_OUTPUT
+      - name: Release
+        uses: softprops/action-gh-release@v2
```
Comment (Stefterv) [inline .github/workflows/release.yml:31]: Great work! 🎉
 just a quick note: `action-gh-release` supports file globbing so `files: release/*.zip` would probably work and results in easier to read code

Comment (enkatsu) [reply .github/workflows/release.yml:31]: Thank you for the suggestion, @Stefterv ! You're right, using globbing would make it easier to understand. I’ll incorporate that 👍

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-version: 8.5
+      - name: Build with Gradle
+        id: build
+        run: gradle packageRelease
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
```
Comment (Stefterv) [inline .github/workflows/release.yml:30]: Sorry just noticed two more things

This if is double with the `on:push:tags` on line 3

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Stefterv) [inline .github/workflows/release.yml:32]: I'm not entirely sure if all of them, but by my knowledge actions nowadays have default access to the `GITHUB_TOKEN` so this is no longer required.

File: .github/workflows/release.yml (hunk 2 above)
Comment (enkatsu) [reply .github/workflows/release.yml:30]: Thank you for the additional advice! I've kept `on:push:tags` and removed the `if:`.

File: .github/workflows/release.yml (hunk 3 above)
Comment (enkatsu) [reply .github/workflows/release.yml:32]: Thank you for the review ! I wrote this code based on older knowledge. After testing in my repository by removing `token: ${{ secrets.GITHUB_TOKEN }}`, the release worked fine, so I remove this line.

**Review by Stefterv — APPROVED** [review]

**Comment by mingness** [comment]
Great work @enkatsu ! Thanks for the review @Stefterv ! 
:pray: 
Will merge.

**Comment by SableRaf** [comment]
Fixed https://github.com/mingness/processing-library-template/issues/9

**Pull request merged**